### PR TITLE
Add validate OpenTelemetry deployment

### PR DESCRIPTION
**Pull request by pavolloffay**
The preview is available at https://github.com/openshift/openshift-docs/pull/63349

Issue:

* https://issues.redhat.com/browse/TRACING-3260 - validating / verification
* https://issues.redhat.com/browse/TRACING-3506 - installation

### CODE REVIEW

File: modules/distr-tracing-install-otel-operator.adoc [hunk 1]
```diff
@@ -52,3 +52,12 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
 . Navigate to *Operators* -> *Installed Operators*.
 
 . On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {OTELName} Operator shows a status of "Succeeded" before continuing.
+
+.Verification
```
Comment (pavolloffay) [inline modules/distr-tracing-install-otel-operator.adoc:102]: This is similar to https://63349--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing

**Comment by pavolloffay** [comment]
@max-cx could you please review?

File: modules/distr-tracing-install-otel-operator.adoc [hunk 2]
```diff
@@ -52,3 +52,12 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
 . Navigate to *Operators* -> *Installed Operators*.
 
 . On the *Installed Operators* page, select the `openshift-operators` project. Wait until you see that the {OTELName} Operator shows a status of "Succeeded" before continuing.
```
Comment (max-cx) [inline modules/distr-tracing-install-otel-operator.adoc]:
```suggestion
. In *Installed Operators*, select the *openshift-operators* project and wait for the {OTELName} Operator status to turn to *Succeeded*.
```
BTW, the Operator is literally listed as **Red Hat OpenShift distributed tracing data collection Operator** in the Operator Hub, correct?

File: modules/distr-tracing-install-otel-operator.adoc (hunk 1 above)
Comment (max-cx) [reply modules/distr-tracing-install-otel-operator.adoc:102]: Thank you for the heads up. I will keep this in mind that this procedure is similar to Tempo installation.

File: modules/distr-tracing-install-otel-operator.adoc [hunk 3]
```diff
@@ -52,3 +52,12 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
...
+
+.Verification
+
+. Verify that the `status.phase` of  OpenTelemetry collector pod is `Running` and the `conditions` are `type: Ready`.
```
Comment (max-cx) [inline modules/distr-tracing-install-otel-operator.adoc:104]: @pavolloffay, verify where exactly?

File: modules/distr-tracing-install-otel-operator.adoc [hunk 4]
```diff
@@ -52,3 +52,12 @@ The *Manual* approval strategy requires a user with appropriate credentials to a
...
+
++
+---
+oc get pod -l app.kubernetes.io/managed-by=opentelemetry-operator,app.kubernetes.io/instance=<namespace>.<instance name> -o yaml
+---
```
Comment (max-cx) [inline modules/distr-tracing-install-otel-operator.adoc]:
```suggestion
[source,terminal]
----
$ oc get pod -l app.kubernetes.io/managed-by=opentelemetry-operator,app.kubernetes.io/instance=<namespace>.<instance name> -o yaml
----
```

File: modules/distr-tracing-install-otel-operator.adoc (hunk 2 above)
Comment (pavolloffay) [reply modules/distr-tracing-install-otel-operator.adoc]: Yes we should change it here

File: modules/distr-tracing-install-otel-operator.adoc (hunk 3 above)
Comment (pavolloffay) [reply modules/distr-tracing-install-otel-operator.adoc:104]: The verification should follow after collector CR installation which is missing in this document.

Comment (max-cx) [reply modules/distr-tracing-install-otel-operator.adoc:104]: What do you want me to do now? Merge this PR or wait until (I assume) you add the collector CR installation step(s)?

Comment (pavolloffay) [reply modules/distr-tracing-install-otel-operator.adoc:104]: It was not on my list but I can add section for it as well

Comment (max-cx) [reply modules/distr-tracing-install-otel-operator.adoc:104]: Is this task part of https://issues.redhat.com/browse/TRACING-3506 or which one?

Comment (max-cx) [reply modules/distr-tracing-install-otel-operator.adoc:104]: Or should I just copy the lines from https://63349--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing to this file?

File: modules/distr-tracing-install-otel-operator.adoc (hunk 1 above)
Comment (max-cx) [reply modules/distr-tracing-install-otel-operator.adoc:102]: Should I just copy the lines from https://62410--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing to this file? `[Yes/No]`

File: modules/distr-tracing-install-otel-operator.adoc (hunk 3 above)
Comment (pavolloffay) [reply modules/distr-tracing-install-otel-operator.adoc:104]: It's in the verification / validation so it is https://issues.redhat.com/browse/TRACING-3260

File: modules/distr-tracing-install-otel-operator.adoc (hunk 1 above)
Comment (pavolloffay) [reply modules/distr-tracing-install-otel-operator.adoc:102]: No, it was just heads up, nothing needs to be copied

**Pull request merged**